library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
+    private boolean disableBloomFilter = disableBloomFilterByDefault;
```

Comment:
I treat enable bloom filter as the default. that's why use disableBloomFilter for special case